clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -261,6 +264,29 @@ option_init(int argc, const char *argv[])
             return false;
         }
     }
+
+    if (!op_elf.specified() && !op_primary_sb.specified()) {
+        std::cerr << CLIENT_NAME << ": at least one of the options --elf, --primary_sb."
```

Comment:
Incomplete statement? Add "must be specified" in the end.

Reply:
Done.